cmd/messages_text.go

```diff
@@ -242,6 +254,13 @@ func (mt *MessagesText) _select(name string) {
 					mt.selectedMessage = i
 				}
 			}
+		} else if ref := ms[mt.selectedMessage].Reference; ref != nil {
+			for i, m := range ms {
+				if ref.MessageID == m.ID {
+					mt.selectedMessage = i
+					break;
+				}
+			}
```

Comment:
Selecting pinned message should have its own keybinding.

Reply:
added 'selectPin' keybinding. Let me know if it should be renamed.